package.json

```diff
@@ -3,12 +3,12 @@
   "version": "0.1.2-alpha.161",
   "description": "",
   "author": "slapshot@yext.com",
-  "main": "./lib/esm/index.js",
-  "types": "./lib/esm/index.d.ts",
+  "main": "./lib/esm/src/index.js",
```

Comment:
the baseUrl for the typescript output was moved down a level to be able to include the tailwind.config.js in the output. This results in all other files being nested in a "src" folder. 
This is like what happened when we included the package.json in our other repos for the SDK Agent work, so that we could access the package json version in source code.

Reply:
I see. I find that somewhat confusing. When I see a directory called `src` under `lib`, I expect it to be the raw source code. This is something I have seen in other npm packages. But, that's not really the case here. It's transpiled source code (from TS to JS).